packages/web-components/src/globals/services-store/actions/searchAPI.ts

```diff
@@ -0,0 +1,119 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { ThunkAction } from 'redux-thunk';
+import { loadLanguage } from './localeAPI';
+import { SEARCH_API_ACTION, SearchAPIState } from '../types/searchAPI';
+
+/**
+ * @param language The locale.
+ * @param searchQueryString The search query string.
+ * @returns The endpoint URL for the given locale and search query string.
+ */
+function getSearchEndpoint(language: string, searchQueryString: string) {
+  const [primary, country] = language!.split('-');
+  return `https://www-api.ibm.com/search/typeahead/v1?lang=${primary}&cc=${country}&query=${searchQueryString}`;
```

Comment:
The host is configurable via environment variable, wondering if we shouldn't be hardcoding it here and instead have a default value if not configured? https://github.com/carbon-design-system/ibm-dotcom-library/blob/14acb43fdc01937dc465f8ae2f768499465980d6/packages/services/src/services/SearchTypeahead/SearchTypeahead.js#L16

Reply:
Makes sense @jeffchew - Updating:

```suggestion
  return `${(process && process.env.SEARCH_TYPEAHEAD_HOST) || 'https://www-api.ibm.com'}/search/typeahead/v1?lang=${primary}&cc=${country}&query=${searchQueryString}`;
```